week1-blog/style.css

```
@@ -0,0 +1,302 @@
+* {
+    box-sizing: border-box;
+}
+
+html {
+    scroll-behavior: smooth;
+}
+
+body {
+    background-color:#93B5C6;
+    margin: 0;
+    padding: 0;
+    overflow-x: hidden;
+}
+
+header {
+    background-color: #FFE3E3;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-direction: row;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    z-index: 1; /* 최상단에 위치 */
+}
+
+#logo {
+    width: 30px;
+    height: 30px;
+    margin-left: 15px;
+}
+
+#nav-btn {
+    width: 15px;
+    height: 15px;
+    margin-right: 15px;
+}
+
+header h2 {
+    font-size: large;
+}
+
+li {
+    list-style: none;
+}
+
+/*컨테이너*/
+.container {
+    display: flex;
+    width: 100%;
+    height: auto;
+    min-height: 100%;
+    padding: 30px 30px 50px 30px;
+    margin-top: 50px;
+}
+
+/*사이드바*/
+aside {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    
+    position: fixed;
+    top: 100px;
+    background-color: #f4f4f4;
+    width: 160px;
+    padding: 10px 20px;
+    border-radius: 15px;
+}
+
+aside h3 {
+    margin: 20px 0 0 0;
+}
+
+.category-content {
+    border-radius: 20px;
+    border: 1px solid black;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 0;
+    width: 100%;
+}
+
+.category-item {
+    width: 100%;
+    text-align: center;
+    padding: 4px 0;
+
+}
+
+.category-item:first-child {
+    border-top-left-radius: 20px;
+    border-top-right-radius: 20px;
```

우앗 이런 속성도 있군요 !! 📝

---

```suggestion
    border-radius: 0 20px;
```
이렇게 더 간단하게 표현할 수도 있습니다!